charms/kserve-controller/tests/integration/test_charm.py

```diff
@@ -147,8 +147,14 @@ async def test_build_and_deploy(ops_test: OpsTest):
     assert ops_test.model.applications[APP_NAME].units[0].workload_status == "active"
 
 
+@pytest.mark.parametrize(
+    "inference_file",
+    ["./tests/integration/sklearn-iris.yaml", "./tests/integration/pmml-server.yaml"],
+)
 @pytest.mark.parametrize("cleanup_namespaces_after_execution", ["raw-namespace"], indirect=True)
-def test_inference_service_raw_deployment(cleanup_namespaces_after_execution, ops_test: OpsTest):
+def test_inference_service_raw_deployment(
```

Comment:
you can create the namespace with a fixture rather than creating it and deleting it for every parameterized run